glui_textbox.cpp

```diff
@@ -180,8 +180,9 @@ int    GLUI_TextBox::key_handler( unsigned char key,int modifiers )
   /*  has_selection = (sel_start != sel_end);              */
 
   if ( key  == CTRL('[')) {         /* ESCAPE */
-    glui->deactivate_current_control();
-    return true;
+    
+	  //GLUI_Main::keyboard returns control to the container.
+	  assert(0); 
```

Comment:
In many ways it's there to explain why the Esc code was deleted. Code elimination is important in many cases. That is "dead code" is not useless code. But if you feel that the new handling of Esc in the top-level window is such an irrevocably great idea that it needs no traces left behind, by all means do so. (My location bar depends on this new Esc feature.)